packages/medusa/src/services/idempotency-key.ts

```diff
@@ -114,7 +115,7 @@ class IdempotencyKeyService extends TransactionBaseService<IdempotencyKeyService
         new Date(key.locked_at).getTime() > Date.now() - KEY_LOCKED_TIMEOUT
 
       if (isLocked) {
-        throw new MedusaError("conflict", "Key already locked")
+        throw new MedusaError(MedusaError.Types.CONFLICT, "Key already locked")
```

Comment:
todo: Add case to `error-handler.ts` and respond with 409 similar to the transactional errors